pam/cmdline_linux.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2023 Yahoo Inc.
+// Licensed under the terms of the Apache License 2.0. Please see LICENSE file in project root for terms.
+
+package pam
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+)
+
+func getCmdLine() []byte {
+	cmd, err := ioutil.ReadFile(fmt.Sprintf("/proc/%d/cmdline", os.Getpid()))
```

Comment:
can we use https://pkg.go.dev/os#ReadFile instead?

Reply:
This was copied from `pam_sshca.go` and I didn't want to change anything.